typing/typedecl_variance.ml

```diff
@@ -148,7 +155,7 @@ let compute_variance_type env ~check (required, loc) decl tyl =
         let var = get_variance ty tvl in
         let (co,cn) = get_upper var and ij = mem Inj var in
         if Btype.is_Tvar ty && (co && not c || cn && not n || not ij && i)
-        then raise (Error(loc, Bad_variance (!pos, (co,cn,ij), (c,n,i)))))
+        then raise (Error(loc, Bad_variance (Variance_not_satisfied !pos, (co,cn,ij), (c,n,i)))))
```

Comment:
This line goes over 80 columns (this explains the Travis failure on [the check-typo test](https://travis-ci.org/ocaml/ocaml/jobs/569923489)), could you fix this by breaking it at an appropriate place?

Reply:
Sure - thanks.